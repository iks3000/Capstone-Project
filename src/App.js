import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import Home from "./pages/Home";
import Inner from "./pages/Inner";

function App() {
  return (
    <Router>
      <Switch>
        <Route exact path="/" component={Home} />
        <Route exact path="/inner" component={Inner} />
      </Switch>
    </Router>
  );
}

export default App;
