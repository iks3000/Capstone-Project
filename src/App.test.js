import { shallow } from "enzyme";
import App from "./App";

describe("<App />", () => {
  it("Renders <App /> component correctly", () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });
});
