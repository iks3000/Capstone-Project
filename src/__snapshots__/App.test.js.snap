// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Renders <App /> component correctly 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/inner"
    />
  </Switch>
</BrowserRouter>
`;
